tests/ops/api/v1/endpoints/test_privacy_request_endpoints.py

```diff
@@ -2925,6 +2925,46 @@ def test_invalid_code(
         )
         assert not mock_dispatch_message.called
 
+    @mock.patch(
+        "fides.api.ops.api.v1.endpoints.privacy_request_endpoints.dispatch_message_task.apply_async"
+    )
+    def test_too_many_attempts(
+        self,
+        mock_dispatch_message,
+        db,
+        api_client,
+        url,
+        privacy_request,
+        privacy_request_receipt_notification_enabled,
+    ):
```

Comment:
Could you add a test for the flow we were seeing in the UI, that you can verify you identity and then save your preferences and your token isn't expired?